### PR TITLE
Ping host workflow

### PR DESCRIPTION
**Pull request by raskin-aleksandr**
Adding ping operation
Ping host workflow
Mail system properties file

### CODE REVIEW

File: org/openscore/slang/base/network/ping.sl [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    - timeout:
+        default: "'1000'"
+        required: false
+
+  action:
+    python_script: |
+        try:
+          import os, smtplib
+          if os.path.isdir('/usr'):
+            response = os.system("ping -c 1 -t " + ttl + ' -s ' + size + ' -W ' + timeout + ' ' + address)
```
Comment (meirwah) [inline org/openscore/slang/base/network/ping.sl]: this works on both win and linux? did you test?

File: org/openscore/slang/base/network/ping.sl [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+          else:
+            response = os.system("ping -n 1 -i " + ttl + ' -l ' + size + ' -w ' + timeout + ' ' + address)
+          if response == 0:
+            message = (address, 'is up!')
```
Comment (meirwah) [inline org/openscore/slang/base/network/ping.sl]: please add standard ping msg...

File: org/openscore/slang/base/network/ping_hosts.sl [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+# - address - list of ip's to be checked
+#
+# For correct work of email part - system_property (mail.yaml) file should be filled and called as '--spf' input
+#
+# Results:
+#  SUCCESS: addressee will get an email with result
+#  FAILURE: addressee will get an email with exception of operation
+# 
+####################################################
+namespace: org.openscore.slang.base.network
```
Comment (meirwah) [inline org/openscore/slang/base/network/ping_hosts.sl]: put this flow under example folder, like : 
org.openscore.slang.base.network.example

File: org/openscore/slang/base/network/ping.sl (hunk 1 above)
Comment (raskin-aleksandr) [inline org/openscore/slang/base/network/ping.sl]: yes, ping operation works on windows and linux as well

File: org/openscore/slang/base/network/ping.sl (hunk 2 above)
Comment (raskin-aleksandr) [inline org/openscore/slang/base/network/ping.sl]: You mean like this:
40 bytes from 16.60.162.31: icmp_seq=1 ttl=64 time=0.027 ms
?

Comment (meirwah) [inline org/openscore/slang/base/network/ping.sl]: add this line : 
40 bytes from 16.60.162.31: icmp_seq=1 ttl=64 time=0.027 ms

as another result.. like "ping_response"

**Pull request closed without merging**